src/Components/CommentAdder.jsx

```diff
@@ -0,0 +1,45 @@
+import { postComment } from "../../utils";
+import { useState } from "react";
+import { useParams } from "react-router";
+
+function CommentAdder({setArticleComments, currentUser, setFetchedComments}) {
+
+    const [isLoading, setIsLoading] = useState(true)
+const {username} = currentUser
+const [newComment, setNewComment] = useState("")
+const [commentMessage, setCommentMessage] = useState("")
+const {article_id }= useParams()
+ 
+function handleSubmit(event) {
+    setFetchedComments(false)
+    event.preventDefault()
+    postComment(username, newComment, article_id).then((newCommentFromApi) => {
+        setNewComment("")
+        setFetchedComments(true)
+        setCommentMessage("Comment posted successfully.")
+        setTimeout(() => {
+            setCommentMessage("")
+        }, 3000)
+         setIsLoading(false)
+        setArticleComments((currComments) => {
+            return [newCommentFromApi, ...currComments]
+        })
+    })
+
+    if (isLoading) return <p>Loading ...</p>
+}
```

Comment:
A message should be displayed that a comment is being posted.

At the moment we can only see a weird message:
<img width="409" alt="Screenshot 2023-06-01 at 16 10 36" src="https://github.com/Tomoneill98/fe-nc-news/assets/54496822/b67457d9-37d5-49d9-bc29-e246d83ddbd4">